cellpack/autopack/IOutils.py

```diff
@@ -567,180 +568,12 @@ def addCompartments(env, compdic, i, io_ingr):
                         # setup recipe
                     o.setInnerRecipe(rMatrix)
 
-                    # put here the export/import ?
 
-
-def save_asJson(env, setupfile, useXref=True, indent=True):

```

Comment:
Love to see it go. What does `mixed` refer to in the one we're going with below? We can probably just call it `save_json` or something.